examples/screen_cast_gstreamer.rs

```diff
@@ -0,0 +1,81 @@
+use ashpd::desktop::PersistMode;
+use ashpd::desktop::screencast::{CursorMode, Screencast, SourceType, Stream};
+use gst::prelude::*;
```

Comment:
You need to add gstreamer to the dev-dependencies

Reply:
Done